src/get-started.md.njk

```diff
@@ -0,0 +1,95 @@
+---
+layout: layout-single-page-prose.njk
+---
+
+# Get started
+
+{% from "_example.njk" import example %}
+
+This guide explains how to create prototypes using the GOV.UK Design System and GOV.UK Prototype Kit.
+
+## Before you start
+
+To make prototypes you will need to download and install a new version of the GOV.UK Prototype Kit which has been built to work with the Design System while it’s in private beta.
+
+The installation process is the same as for the current version. You should:
+
+* [download the new version of the GOV.UK Prototype Kit](https://govuk-prototype-kit-beta.herokuapp.com/)
+* [follow these installation instructions](https://govuk-prototype-kit-beta.herokuapp.com/docs/install/introduction)
+
+The rest of the guide assumes that you are familar with the GOV.UK Prototype Kit.
+
+## Using the new Prototype Kit
+
+The new version of the Prototype Kit is identical to the one you may have used before except that it uses a new frontend framework called GOV.UK Frontend.
+
+This means that any code that you copy across from old prototypes will not display correctly. Instead, you should use the code provided in the Design System.
+
+## Styling content
+
+The Design System provides lots of new CSS classes for styling content, meaning you shouldn’t need to write your own Sass or CSS.
```

Comment:
I don't think this is true… I think they'd hopefully need to write _less_ Sass or CSS? Prototype kit users might still want to create things that can't be done using the components we've got so far.

Reply:
Requires review from @timpaul and @amyhupe

Reply:
Change:

The Design System provides lots of new CSS classes for styling content, meaning you shouldn’t need to write your own Sass or CSS.

To:

The Design System provides lots of new CSS classes for styling content, meaning you shouldn’t need to write as much of your own Sass or CSS.

Reply:
Done